website/tutorial/part_1/03_installation.md

```diff
@@ -11,68 +11,32 @@
+ * Git - [https://git-scm.com/]()
+ * Docker - [https://www.docker.com]()
 
-To install the SDK manually:
+## General installation instructions
 
-* install [Homebrew](http://brew.sh)
-  * run `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
-  * then run `brew doctor` and fix all the errors
-* install [Node.js](https://nodejs.org) version 6 or above: `brew install node`
-* install Exosphere: `npm i -g exosphere-sdk`
-* verify that you can run Exosphere commands: `exo version`
+* Download and install the binary executable for your platform from the 
+[GitHub release page](https://github.com/Originate/exosphere/releases/latest)
+* Move the binary to a location in your path and name it `exo`
 
+### Mac installation
 
-## Windows
-
-### Windows 7
-
-Some of the commands below have to be run in an __administrative command shell__.
-Instructions for how to open such a shell are available
-for [windows 8-10](http://www.howtogeek.com/194041/how-to-open-the-command-prompt-as-administrator-in-windows-8.1)
-and [Windows 7](http://www.howtogeek.com/howto/windows-vista/run-a-command-as-administrator-from-the-windows-vista-run-box)).
-
-* install the package manager
-  * in an administrative shell, install [Chocolatey](https://chocolatey.org/install)
-  * when done, close the current shell and open a new one to load the environment changes prepared by the installer
-* install [Node.js](http://nodejs.org) version 6 or above
-  * in another administrative shell: `choco install nodejs.install -y`
-* install Exosphere:
-  * in a normal shell: `npm install --global exosphere-sdk`
-  * close this shell, open a normal one, and run `exo` to make sure it works
-* install other tools you will need
-  * in an administrative shell: [Git](https://git-scm.com) (`choco install git.install -y`
-  * optionally [Github Desktop](https://desktop.github.com) (install manually, the choco package is broken)
-
-
-### Contributing
-
-If you want to become an Exosphere contributor and you are on Windows,
-you need some additional infrastructure:
-* [Git](https://git-scm.com) for the Git command line and Git Bash: `choco install git.install -y`
-* you also need to set up SSH keys for Github.
-  The easiest way is via [Github Desktop](https://desktop.github.com): `choco install github -y`
-  (Note: this install is currently broken, you might want to install this manually)
-
-Please perform all
-
-
-
-## Linux
-
-Installation instructions given for [Ubuntu](http://www.ubuntu.com),
-please adapt them to your distro as needed:
-* install [ZeroMQ](http://zeromq.org)
-* install [Node.js](https://nodejs.org) version 6 or above: `sudo apt-get install node`
+_Note: Depending on the privileges of `/usr/local/bin` on your system these steps could require root privileges_
 
+In a terminal, execute the following:
+```bash
+curl -L https://github.com/Originate/exosphere/releases/download/vX.Y.Z/exo-darwin-amd64 >/usr/local/bin/exo
```

Comment:
Not sure how much easier it is to provide a curl command here. Users have to look up the version manually on the website anyways - why not have them download the correct binary right there?

We could tell them to move the binary to `/usr/local/bin` if they don't have a good other place to put it.